package.json

```
@@ -43,7 +43,7 @@
     "lint": "eslint shared app src --ext .ts",
     "list-outdated-deps": "npm out; cd app && npm out; true",
     "prepare": "cd app && npm ci && cd .. && npm run build",
-    "relock": "rm -rf ./node_modules/ ./app/node_modules/ ./npm-shrinkwrap.json ./app/npm-shrinkwrap.json; npm install --ignore-scripts --package-lock && mv package-lock.json npm-shrinkwrap.json && npm out; cd app && npm install --ignore-scripts --package-lock && mv package-lock.json npm-shrinkwrap.json && npm out; cd .. && true",
+    "relock": "rm -rf ./node_modules/ ./app/node_modules/ ./npm-shrinkwrap.json ./app/npm-shrinkwrap.json && npm install --ignore-scripts --package-lock && mv package-lock.json npm-shrinkwrap.json && npm out && cd app && npm install --ignore-scripts --package-lock && mv package-lock.json npm-shrinkwrap.json && npm out",
```

@TheCleric hmmm, I understand `true` doesn't exist under Windows CMD, but I thought `;` does and has the same behavior as Unix's `;`, that is to keep chaining regardless of error code. Said differently, doesn't this work?

```suggestion
    "relock": "rm -rf ./node_modules/ ./app/node_modules/ ./npm-shrinkwrap.json ./app/npm-shrinkwrap.json; npm install --ignore-scripts --package-lock && mv package-lock.json npm-shrinkwrap.json && npm out; cd app && npm install --ignore-scripts --package-lock && mv package-lock.json npm-shrinkwrap.json && npm out; cd ..",
```

---

When I get on my window machine I will give that a shot. Maybe it was just the true. Unfortunately all I was really getting was something "that didn't work and it's not our fault" out of npm.

---

Also I tried just using the `npm shrinkwrap` command, but that seems to foul up the `fs-events` stuff in the shrinkwrap, which is the reason I assume you had avoided it.

---

> When I get on my window machine I will give that a shot. Maybe it was just the true. Unfortunately all I was really getting was something "that didn't work and it's not our fault" out of npm.

@TheCleric 👍

> Also I tried just using the `npm shrinkwrap` command, but that seems to foul up the `fs-events` stuff in the shrinkwrap, which is the reason I assume you had avoided it.

@TheCleric yup; if you look at commits from last week you'll see a few commits of me fumbling around `fs-events` nonsense in CI due to differences between `npm install --package-lock`'s and `npm shrinkwrap`. Thus, the current situation :shrug: